### PR TITLE
Cleanup and restructure API reference overview

### PR DESCRIPTION
**Pull request by morphis**
This drops things which do not belong in there / or do not belong together.

The dropped API overview for AMS will move into the swagger where it belongs.

### CODE REVIEW

**Comment by keirthana** [comment]
> The dropped API overview for AMS will move into the swagger where it belongs.

@jat-canonical @morphis Are there PRs or Jira issues tracking the overview content move to swagger. I am asking so that we don't lose track of this once I merge this PR

File: reference/api-reference.md [hunk 1]
```diff
@@ -1,194 +1,29 @@
 (ref-api)=
 # API reference
 
-All communication between AMS and its clients happen using a RESTful API over HTTP. This API is encapsulated over either TLS (for remote operations) or a Unix socket (for local operations).
+Anbox Cloud includes various APIs to help you building solutions with it. This page provides an overview over the available APIs and what they are used for.
```
Comment (keirthana) [inline reference/api-reference.md:4]:
```suggestion
Anbox Cloud includes various APIs to help you build solutions with it. This page provides an overview of the available APIs and their uses.
```

File: reference/api-reference.md [hunk 2]
```diff
@@ -1,194 +1,29 @@
...
+## Available APIs
 
-* AMS HTTP API
-* Anbox Cloud HTTP API
-* Stream Gateway API
-* Anbox Platform SDK API
+### AMS HTTP API
 
-All these APIs except for the {ref}`anbox-https-api` have an auto-generated specification describing its API endpoints.
+The AMS services provide a HTTP based API to allow you managing all of its objects. Internally the `amc` client uses the same API to perform its operation.
```
Comment (keirthana) [inline reference/api-reference.md:12]: vowel sound, so an HTTP
```suggestion
The AMS services provide an HTTP based API to help you manage all of its objects. Internally the `amc` client uses the same API to perform its operation.
```

File: reference/api-reference.md [hunk 3]
```diff
@@ -1,194 +1,29 @@
...
-## Authentication
+You can find the OpenAPI specification [here](https://canonical.github.io/anbox-cloud.github.com/latest/ams/).
 
-Not all REST API endpoints require authentication, for example, the following API calls are allowed for everyone:
+### Anbox HTTP API
 
-* `GET /`
-* `GET /1.0`
-* `GET /1.0/version`
+The Anbox runtime inside each instance managed by AMS provides a HTTP API over a UNIX domain socket to allow interacting with various functionality Anbox provides. This for example includes the injection of position or sensor events into Android.
```
Comment (keirthana) [inline reference/api-reference.md:18]:
```suggestion
The Anbox runtime inside each instance managed by the AMS provides an HTTP API over a UNIX domain socket that helps interacting with various functionalities offered by Anbox. This includes the injection of position or sensor events into Android, among other things.
```

File: reference/api-reference.md [hunk 4]
```diff
@@ -1,194 +1,29 @@
...
 
-Some endpoints require an additional authentication token to ensure that the requester is authorised to access the resource, for example:
+You can find the API specifiction at {ref}`anbox-https-api`.
 
-* `GET /1.0/artifacts`
-* `PATCH /1.0/instances/<name>`
+### Anbox Stream Gateway API
 
-## API versioning
+The Anbox Stream Gateway service provides an API endpoint to allow you managing streaming sessions.
```
Comment (keirthana) [inline reference/api-reference.md:24]:
```suggestion
The Anbox Stream Gateway service provides an API endpoint to help you to manage streaming sessions.
```

**Review by keirthana — CHANGES_REQUESTED** [review]
Minor comments based on the Ubuntu style guide and one question about moving the overview to swagger.
Otherwise, LGTM!

**Comment by morphis** [comment]
@keirthana yes, see https://github.com/canonical/ams/pull/495

**Review by keirthana — APPROVED** [review]
LGTM, if spellcheck issues are addressed (@morphis)

**Pull request merged**